lib/av_client.ts

```diff
@@ -510,6 +513,18 @@ export class AVClient implements IAVClient {
     return ballotConfigs[voterSession.content.voterGroup]
   }
 
+  public getVoterContestConfigs(): ContestConfig[] {
```

Comment:
Where is this method used?

Reply:
It is not used in JS-client, but I asked @av-mads yesterday, and he told me that it is supposed to be used by calling it from outside (instances of AVclient I'd guess).

Reply:
It's used by liminal I think. May also be used elsewhere